cpp_test_suite/old_tests/lock.cpp

```diff
@@ -8,6 +8,8 @@
 #ifdef WIN32
 #include <process.h>
 #define WEXITSTATUS(w) w
+#elif defined(__OpenBSD__)
+#include <sys/wait.h>
 #else
 #include <wait.h>
```

Comment:
On Debian and Centos all `wait.h` contains is:
```
+(michal@b7fefd8ccd7f) /usr/include $ cat wait.h
#include <sys/wait.h>
```

```suggestion
#else
#include <sys/wait.h>
```